diff --git a/cmd/bonusly/bonusly.go b/cmd/bonusly/bonusly.go
index b8a477c..54abea5 100644
--- a/cmd/bonusly/bonusly.go
+++ b/cmd/bonusly/bonusly.go
@@ -9,7 +9,7 @@ import (
 
 	bonusly "github.com/kimchelly/go-bonusly"
 	"github.com/pkg/errors"
-	"github.com/urfave/cli"
+	cli "github.com/urfave/cli/v2"
 )
 
 func main() {
diff --git a/go.mod b/go.mod
index 25f5779..b7c28c1 100644
--- a/go.mod
+++ b/go.mod
@@ -6,5 +6,5 @@ require (
 	github.com/PuerkitoBio/rehttp v1.0.0
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.6.1
-	github.com/urfave/cli/v2 v2.2.0 // indirect
+	github.com/urfave/cli/v2 v2.2.0
 )
